FirstTask.py

```diff
@@ -0,0 +1,230 @@
+            if code == response.pass_code():
+                self.logged_in_account = response
+                return True, "Successfully Login!"
+        return False, f"{account_number} Account Not Found!"
+
+    def transfer_money(self, sender_account_number, receiver_account_number, amount):
+        status_1, response_1 = self.find_account(sender_account_number)
+        status_2, response_2 = self.find_account(receiver_account_number)
+        if status_1 and status_2:
+            if response_1.draw_money(amount):
+                response_2.deposit_bill(amount)
+                return True, "Money Successfully transferred!"
+        else:
+            if status_1 == False:
+                account_not_found = sender_account_number
+            elif status_2 == False:
+                account_not_found = receiver_account_number
+            if status_1 == False and status_2 == False:
+                return False, f"Both {sender_account_number} and {receiver_account_number} are invalid!"
+            return False, f"{account_not_found} Account Not Found!"
+    
+    def compare(self, current_account, other_account):
+        status_1, response_1 = self.find_account(current_account)
+        status_2, response_2 = self.find_account(other_account)
+        if status_1 and status_2:
+            if response_1.check_balance() > response_2.check_balance():
+                return True, "Your Account is Greater than Provided Account!"
+            if response_1.check_balance() < response_2.check_balance():
+                return True, "Your Account is Smaller than Provided Account!"
+            if response_1.check_balance() == response_2.check_balance():
+                return True, "Your Account is Equal to the Provided Account!"
+        else:
+            if status_1 == False:
+                account_not_found = current_account
+            elif status_2 == False:
+                account_not_found = other_account
+            if status_1 == False and status_2 == False:
+                return False, f"{current_account} and {other_account} are invalid!"
+            return False, f"{account_not_found} Account Not Found!"
+   
+    def account_details(self, account_number):
+        status,response = self.find_account(account_number)
+        if status:
+            response.account_details()
+        else:
+            return False, f"{account_number} Account Not Found!"
+   
+    def logout(self):
+        self.logged_in_account = None
+        return self.logged_in_account
+
+    def is_logged_in(self):
+        if self.logged_in_account:
+            return True
+        return False
+
+def run():
+    bank = Bank()
+    while(True):
+        option = int(input("1) Create Account \n2) Login \n3) Find Account \n4) Total Accounts\n0) Exit \nChoose:"))
+        if option == 1:
+            sys('clear')
+            name = input("Enter Full Name: ")
+            pass_code = int(input("Enter Passcode "))
+            while(True):

```

Comment:
One improvement here is to define `account_select` before the `while` loop and adding the break condition like this:
```
account_select = None
while not account_select:
```
Then you can just remove the `break` statement and the loop would continue until the user selects a valid type.